dids/src/main/kotlin/web5/sdk/dids/ion/model/Models.kt

```diff
@@ -102,7 +102,10 @@ public enum class PublicKeyPurpose(@get:JsonValue public val code: String) {
 )
 @JsonSubTypes(
   JsonSubTypes.Type(AddServicesAction::class, name = "add-services"),
-  JsonSubTypes.Type(ReplaceAction::class, name = "replace")
+  JsonSubTypes.Type(ReplaceAction::class, name = "replace"),
+  JsonSubTypes.Type(RemoveServicesAction::class, name = "remove-services"),
+  JsonSubTypes.Type(AddPublicKeysAction::class, name = "add-public-keys"),
+  JsonSubTypes.Type(RemovePublicKeysAction::class, name = "remove-public-keys"),
 )
 public sealed class PatchAction
```

Comment:
Curious why the sealed class route vs a marker interface?

(https://github.com/TBD54566975/web5-kt/pull/77#discussion_r1354476007)